ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -950,14 +951,14 @@ open class DrawerController: UIViewController {
 
 extension DrawerController: UIViewControllerTransitioningDelegate {
     public func animationController(forPresented presented: UIViewController, presenting: UIViewController, source: UIViewController) -> UIViewControllerAnimatedTransitioning? {
-        if presentationStyle(for: source) == .slideover {
+        if presentationStyle(for: source) == .slideover && UIView.areAnimationsEnabled {
             return DrawerTransitionAnimator(presenting: true, presentationDirection: presentationDirection(for: source.view), containerOffset: shadowOffset)
         }
         return nil
     }
 
     public func animationController(forDismissed dismissed: UIViewController) -> UIViewControllerAnimatedTransitioning? {
-        if let controller = dismissed.presentationController as? DrawerPresentationController {
+        if let controller = dismissed.presentationController as? DrawerPresentationController, UIView.areAnimationsEnabled {
```

Comment:
UIView animations are static class not specific to drawers animation. This can have side effects if content view's animations are disabled. 

An appropriate place to do this would be in `dismissWithTransitionContext`

```swift
private func dismissWithTransitionContext(_ transitionContext: UIViewControllerContextTransitioning, completion: @escaping ((Bool) -> Void)) {
        if transitionContext.isAnimated  {
            // cancel or return from transition
        }
}
```

Reply:
I verified this and this doesn't have any effects on using animation blocks. Resolving the comment for now !!